src/main/resources/site/lib/site.js

```diff
@@ -3,32 +3,21 @@ var libs = {
 	util: require('/lib/enonic/util/util')
 };
 
-// "Globals":
 var appNamePropertyName = app.name.replace(/\./g,'-');
 
 function getConfig() {
 	return libs.portal.getSiteConfig();
 }
-function commaStringToArray(string) {
-	var arr = [];
-	string = string.replace(/ /g,''); // Remove all spaces in the string
-	arr = string.split(',');
-	return arr;
+function commaStringToArray(str) {
+	return str.split(',').trim();
```

Comment:
We get a crash here when the setting is not in use. I'm gonna go ahead and change this to handle that.